### PR TITLE
#165273475 Create, view and update user profile

### PR DESCRIPTION
**Pull request by engjames**
**What does this PR do?**
- Creates a user profile

**Description of the task completed**
- Add a user profile
  Before creating a profile, a user should first create an account and login to generate a token. This token grants a user to edit his or her profile.

- Edit user profile
image
--Now I have permissions to edit my profile
image

- View your user profile
 
image

- View other profiles
  
image

- Can not edit other people's profile
 
image

**What are the relevant pivotal tracker stories?**
[#165273475](https://www.pivotaltracker.com/story/show/165273475)

### CODE REVIEW

**Review by mwinel — APPROVED** [review]
This is ready for a merge.

**Review by KabohaJeanMark — APPROVED** [review]
Great job @engjames. I have tested this out and it works well. This is ready for a merge.

File: authors/apps/profiles/urls.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import UserProfiles, Updateprofile
+
+urlpatterns = [
+    path('users/profiles/', UserProfiles.as_view(), name="get_profile"),
+    path('users/profile/<str:username>', Updateprofile.as_view(), name="update_profile")
```
Comment (archibishop) [inline authors/apps/profiles/urls.py]: @engjames Add the slash at the end of the url to follow team conventions.

**Review by archibishop — APPROVED** [review]
@engjames LGTM I am merging this.

**Pull request merged**